bytes-filter.js

```diff
@@ -30,14 +26,11 @@ function factory() {
 
     var units = ['bytes', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB'];
     var number = Math.floor(Math.log(bytes) / Math.log(1024));
-    var value = (bytes / Math.pow(1024, Math.floor(number))).toFixed(precision);
+    var value = (bytes / Math.pow(1024, Math.floor(number))).toFixed(
+      precision);
```

Comment:
Is this over 80 chars or something? Github diff needs to show the 80 char line :)